### DISCUSSION TITLE
Sidebery instead of TreeStyleTabs

### DISCUSSION THREAD

**Discussion by KAGEYAM4** [Floorp-Projects/Floorp#918] [Feature Request]
From sidebery github ->
```
Sidebery is a highly configurable sidebar with panels of different types. Some of the key features:

    Vertical tabs panels with tree or flat layout
    Bookmarks panels
    History panel
    Search in panels
    Customizable context menu
    Customizable styles
    Snapshots (saved windows/panels/tabs)
    ...and more
```

**Comment by Cyanide-1** [comment]
Sounds like a great idea, maybe implement support for both?

**Reply by KAGEYAM4** [reply]
Yes, if they don't affect each other in terms of CSS.

Also, i haven't used TreeStyleTabs, when i started i just looked at the UI and found Sidebery more UI Friendly that's why i started using it. But i would love to know the prespective of someone who uses TreeStyleTabs. Do you use TreeStyleTabs? I will list Some features of Sidebery can you tell me wether TreeStyleTabs has it or something similar.
- Panel ( collection of tabs - similar to workspace, )
- Panel Rueles
- Folders ( just for grouping )
- Color Folder/Tabs, Inhesit color from parent
- Color Based on Tab favicon/type
- Snapshot ( can be automated, it really saved me few times where i lost my tabs )
- Other keyboard/mouse shortcuts for good UX

Here screenshot of my Sidebery ->
- Left vertical is Panel
- Right vertical is Global Pinned Tabs ( maybe in future we will have local/global Pinned Tabs )
- Middle are Tabs in Current panel
![image](https://github.com/Floorp-Projects/Floorp/assets/75798544/381f073b-6076-42bc-a395-270d77a4dbc4)

**Comment by MaxHasBeenUsed** [comment]
Just an update: devs from both sideberry and floorp have confirmed willingness to integrate.

See: https://github.com/mbnuqw/sidebery/issues/1416

**Comment by notDavid** [comment]
Fyi, this may be interesting for some people stumbling upon this, for a somewhat better integration with Sidebery and the sidebar: https://github.com/soulhotel/FF-ULTIMA

Note that i disabled everything in FF-ultima except:

```css
@import url(FF-ULTIMA/theme/all-global-positioning.css);
@import url(FF-ULTIMA/theme/function-sidebar-autohide.css);
@import url(FF-ULTIMA/theme/function-sidebery-autohide.css);
```
